Kernel/Bus/USB/UHCI/UHCIDescriptorPool.h

```diff
@@ -70,7 +70,7 @@ class UHCIDescriptorPool {
     UHCIDescriptorPool(NonnullOwnPtr<Memory::Region> pool_memory_block, StringView name)
         : m_pool_name(name)
         , m_pool_region(move(pool_memory_block))
-        , m_pool_lock(LockRank::None)
+        , m_pool_lock()
```

Comment:
Is there a notable difference here between `m_pool_lock()` and just omitting it entirely? The similar change in UHCIController just removed the entire line.

Reply:
I guess while rebasing I just overlooked some of these.